assets/js/googlesitekit/notifications/register-defaults.js

```diff
@@ -142,6 +148,37 @@ export const DEFAULT_NOTIFICATIONS = {
 		},
 		isDismissible: false,
 	},
+	setup_error: {
+		Component: SetupErrorNotification,
+		priority: 150,
+		areaSlug: NOTIFICATION_AREAS.ERRORS,
+		viewContexts: [ VIEW_CONTEXT_SPLASH ],
```

Comment:
Since this error is specifically only for setup of the plugin, and currently it is showing together with another more general error one, practically shadowing it, never being visible, this one is set to show only on splash screen, while more general error one will be shown on other screens like dashboard etc, so this one will be finally used where intended